crates/swc_ecma_minifier/tests/single-pass/1/output.js

```diff
@@ -6,8 +6,6 @@ var condString, exprAny1, exprBoolean1, exprNumber1, exprString1, exprIsObject1,
     "2",
     "3"
 ];
-condString.toUpperCase;
-array[1];
-condString.toUpperCase;
-array[1];
+condString.toUpperCase, foo(), array[1], foo();
```

Comment:
related issue tersr/terser#1227

We could mark function return literal as pure though